### PR TITLE
Merge/navbar header

### LINKED ISSUE #49: Merge navbar and header sections

### PR DESCRIPTION
**Pull request by abdelrhmanali29**
##What?
Merge Navbar component in the header component

### CODE REVIEW

File: frontend/src/components/Header/index.jsx [hunk 1]
```diff
@@ -15,25 +14,30 @@ const Header = () => {
   }, []);
   if (data) {
     return (
-      <header className=" w-full  bg-c400  border-c100 border-b-4 pb-32">
-        <div className="container flex flex-row pt-64 relative">
+      <header className=" w-full  bg-c400  border-c100 border-b-4 pb-40  pt-8">
+        <Navbar />
```
Comment (Nilegfx) [inline frontend/src/components/Header/index.jsx]: why are we adding the nav "inside" the header gere in this file?  is it hard to add both Navbar and header with one container in `App.js`?
```js
<header className=" w-full  bg-c400  border-c100 border-b-4 pb-40  pt-8">
    <Navbar />
    <Header />
<header />
```
### Another suggestion
both Navbar and Header have the same container in their own components :

`Navbar.js`
```js
<header className=" w-full  bg-c400  border-c100 border-b-4 pb-40  pt-8">
// Navbar code
<header />
```

`Header.js`
```js
<header className=" w-full  bg-c400  border-c100 border-b-4 pb-40  pt-8">
// Header code
<header />
```

and then in `App.js`:
```js
<Navbar />
<Header />
```

**Review by Nilegfx — CHANGES_REQUESTED** [review]

File: frontend/src/components/Header/index.jsx (hunk 1 above)
Comment (MAlngaawy) [reply frontend/src/components/Header/index.jsx]: You are right, I think the first suggestion is the best :+1:

**Review by MAlngaawy — APPROVED** [review]
LGTM

**Review by Nilegfx — APPROVED** [review]
LGTM

**Pull request merged**